modules/cli-developer-other.adoc

```diff
@@ -30,9 +30,15 @@ $ oc plugin list
 
 == version
 
-Display the `oc` client and server versions.
+Display the `oc` client and Kubernetes server versions.
```

Comment:
_maybe_ `Kubernetes API server` instead of Kubernetes server? 

and below: `API server version` instead of server version?
@soltysh WDYT?

Reply:
Let's keep the original wording. It's sufficient.

Reply:
sounds good, thanks

Reply:
Thanks @soltysh - Are you saying that I should remove "Kubernetes" here? Or keep that?

Reply:
Yup, just leave the old wording as it was.

Reply:
Thanks, will do!